.ci/wercker.sh

```diff
@@ -137,12 +137,9 @@ no-error-htmlunit)
   CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
   echo CS_version: ${CS_POM_VERSION}
-  echo "checkouting project sources ..."
-  svn -q export https://svn.code.sf.net/p/htmlunit/code/trunk/htmlunit@r14923 htmlunit
-  cd htmlunit
- sed -i "s/            <version>2.28-SNAPSHOT/            <version>2.28-20171106.080245-12/" pom.xml
-  echo "Running checkstyle validation ..."
-  mvn -e compile checkstyle:check -Dcheckstyle.version=${CS_POM_VERSION}
+  checkout_from https://github.com/HtmlUnit/htmlunit
```

Comment:
as you would like to activate it ..... thanks a lot
but we need to activate shell execution in wercker config - https://github.com/checkstyle/checkstyle/blob/master/wercker.yml#L145

Reply:
Should I update wercker config in this PR?

Reply:
it is better to move htmlunt to separate PR, as it is not required for deprecation issue.

Reply:
Their GitHub repo has 3.1.1, SVN does not, that's why I made the change.

Reply:
you did right update, thank a lot, separate commit is good, but if you change this PR one more time - please move this update to separate PR. Lets not over complicate this PR with unrelated items. We need to complete this PR sooner.